examples/with-markdoc/README.md

```diff
@@ -0,0 +1,59 @@
+# Astro Example: Markdoc (experimental)
+
+This starter showcases the experimental Markdoc integration.
+
+```
+npm create astro@latest -- --template with-markdoc
+```
+
+[![Open in StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/github/withastro/astro/tree/latest/examples/with-markdoc)
+[![Open with CodeSandbox](https://assets.codesandbox.io/github/button-edit-lime.svg)](https://codesandbox.io/p/sandbox/github/withastro/astro/tree/latest/examples/with-markdoc)
+
+> 🧑‍🚀 **Seasoned astronaut?** Delete this file. Have fun!
+
+## 🚀 Project Structure
+
+Inside of your Astro project, you'll see the following folders and files:
+
+```
+/
+├── public/
+├── src/
+│   └── content/
+        └── docs/
+│           └── intro.mdoc
+|       └── config.ts
+│   └── components/
+|       ├── Aside.astro
+│       └── DocsContent.astro
+│   └── layouts/
+│       └── Layout.astro
+│   └── pages/
+│       └── index.astro
+|   └── env.d.ts
+├── astro.config.mjs
+├── README.md
+├── package.json
+└── tsconfig.json
+```
+
+Markdoc can be used in content collections _exclusively_ with the `.mdoc` extension. See `content/docs/` for an example file.
```

Comment:
I wanted to use language here that didn't feel overly discouraging. This makes what you _can't_ do more clear than what you _can_ do. I'd consider removing the "must be located within `src/content/`" piece since it's implied. If you prefer "only" to "exclusively," that's fine!

Reply:
The real problem is just that the adjective is in the wrong place and doesn't actually convey what you mean to convey. If you want it to sound more positive, I can do that! 

(As written, this is says you can "only" use Markdoc in content collections if you use the `.mdoc` extension. That's not the meaning we're going for.)

Reply:
```suggestion
Markdoc (`.mdoc`) files can be used in content collections to author your Markdown content alongside Astro and server-rendered UI framework components (React, Vue, Svelte, and more). See `src/content/docs/` for an example file.
```
What about something like this to replace the two paragraphs?

Reply:
That sounds good to me!